GAMES.json

```diff
@@ -1183,12 +1184,17 @@
 	"323720": true,
 	"324070": true,
 	"324120": true,
+	"324260":
+	{
+		"Comment": "Requires Mono"
```

Comment:
Well, people have to install it manually @johndrinkwater, so why shouldn't it be noted?

@directhex yeah, they know about it, they haven't done anything about it